bugbug/models/bugtype.py

```diff
@@ -194,11 +133,14 @@ def overwrite_classes(
         classes: dict[int, np.ndarray],
         probabilities: bool,
     ):
+        bug_map = {bug["id"]: bug for bug in bugs}
+
         for i, bug in enumerate(bugs):
-            for type_ in bug_to_types(bug):
-                if probabilities:
-                    classes[i][TYPE_LIST.index(type_)] = 1.0
-                else:
-                    classes[i][TYPE_LIST.index(type_)] = 1
+            for j, is_type in enumerate(self.label_extractors):
```

Comment:
I know that I suggested the old name, but I fell these name is more descriptive:
```suggestion
            for j, is_type_applicable in enumerate(self.bug_type_extractors):
```